### PR TITLE
BaSP-MR-48: Members GET-DELETE

### PR DESCRIPTION
**Pull request by francoax**
Development done:
- Display of members list
- Delete button in the list
- Search filter in the list

### CODE REVIEW

**Review by FontanR — APPROVED** [review]
LGTM

File: src/Components/Members/Modal/deletemodal.module.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
+.modal {
+  position: absolute;
+  z-index: 1;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgb(0, 0, 0);
```
Comment (JuanCruzMoreira) [inline src/Components/Members/Modal/deletemodal.module.css]: Please keep consistency in colors language (use hex or rgb throughout the file)

File: src/Components/Members/Toast/toast.module.css [hunk 2]
```diff
@@ -0,0 +1,33 @@
+.toast {
+  position: absolute;
+  display: flex;
+  align-items: center;
+  width: 313px;
+  height: 70px;
+  box-shadow: 0px 10px 23px rgba(0, 0, 0, 0.12);
```
Comment (JuanCruzMoreira) [inline src/Components/Members/Toast/toast.module.css:7]: color name consistency

**Review by JuanCruzMoreira — CHANGES_REQUESTED** [review]

File: src/Components/Members/index.jsx [hunk 3]
```diff
@@ -0,0 +1,100 @@
+import { useEffect, useState } from 'react';
```
Comment (JuanCruzMoreira) [inline src/Components/Members/index.jsx:2]: import "React" too

File: src/Components/Members/index.jsx [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+      console.log(membersFromDb);
+      setMembers(membersFromDb);
+    };
+    getMembers();
+  }, []);
+
+  const getAllMembers = async () => {
+    try {
+      const res = await fetch(`${process.env.REACT_APP_URL_API}/api/member`);
+      console.log(res);
```
Comment (JuanCruzMoreira) [inline src/Components/Members/index.jsx]: Delete this console.log()

**Review by JuanCruzMoreira — CHANGES_REQUESTED** [review]

**Review by JuanCruzMoreira — APPROVED** [review]

**Pull request merged**